src/main.cpp

```diff
@@ -1735,9 +1751,11 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
     bool truncate_inputs = false;
 
     unsigned int flags = fStrictPayToScriptHash ? SCRIPT_VERIFY_P2SH : SCRIPT_VERIFY_NONE;
+    if (protocol_cleanup)
+        flags |= SCRIPT_VERIFY_PROTOCOL_CLEANUP;
```

Comment:
Or
```C++
flags |= -protocol_cleanup & SCRIPT_VERIFY_PROTOCOL_CLEANUP;
```
(without `if` line)

Reply:
Opted against, but added { } brackets for style guide.